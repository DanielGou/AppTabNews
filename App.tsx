import React from 'react';
import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack';

import Home from './src/home/Home'
import { LogoTitle } from './LogoTitle';

const Stack = createNativeStackNavigator();

export default function App() {
  return (
    <NavigationContainer>
      <Stack.Navigator
       screenOptions={{
        headerTitle: ()=>  <LogoTitle/>,
        headerStyle:{
          backgroundColor: '#24292f',
        },
      }}
      >
        <Stack.Screen 
          name="home" 
          component={Home} />
      </Stack.Navigator>
    </NavigationContainer>
  );
}

