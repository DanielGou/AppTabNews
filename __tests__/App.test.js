import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App.tsx';

const tree = renderer.create(<App />);

test('snapshot App', ()=>{
  expect(tree).toMatchSnapshot();
})
