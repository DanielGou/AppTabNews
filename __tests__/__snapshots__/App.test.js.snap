// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot App 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNSScreenStack
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      collapsable={false}
      forwardedRef={[Function]}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onTransitionProgress={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      stackPresentation="push"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backgroundColor="#24292f"
        color="rgb(0, 122, 255)"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={false}
        hideBackButton={false}
        largeTitleHideShadow={false}
        title="home"
        titleColor="rgb(28, 28, 30)"
        topInsetEnabled={false}
        translucent={false}
      >
        <RNSScreenStackHeaderSubview
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
          type="center"
        >
          <Image
            source={1}
            style={
              Object {
                "height": 35,
                "width": 35,
                "zIndex": 10,
              }
            }
          />
          <Text
            style={
              Object {
                "color": "#fff",
                "fontWeight": "bold",
                "marginLeft": 10,
              }
            }
          >
            TabNews
          </Text>
        </RNSScreenStackHeaderSubview>
      </RNSScreenStackHeaderConfig>
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            Object {
              "backgroundColor": "rgb(242, 242, 242)",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "height": "100%",
              "justifyContent": "center",
            }
          }
        >
          <ActivityIndicator
            size="large"
          />
        </View>
      </View>
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
